doc/performance-investigations/0018-pdf-weasyprint-pdfkit/summary.md

```diff
@@ -0,0 +1,113 @@
+|----------------------------------------------------------|---------------------|------------------|-------------------|----------------------|----------------------|----------------------|----------------------|------------------------|-----------------------|-----------------------------------|----------------|----------------|------------------------------------------------------|  
+| Baseline fresh deployment (No PDF download requested)    | 55.41               | 334              | 107               | N/A                  | N/A                  | N/A                  | N/A                  | N/A                    | N/A                   | 14.9                              | 33,246         | 0              | [outputs](outputs/baseline/2022-02-17T14:49:13)      |  
+| Weasyprint with custom CSS file(Run 1, fresh deployment) | 50.91               | 769              | 143               | 1300                 | 2100                 | 2700                 | 7400                 | 9200                   | 10000                 | 38                                | 30,548         | 0              | [outputs](outputs/investigation/2022-02-17T15:17:20) |  
+| Weasyprint with custom CSS file (Run 2)                  | 51.33               | 415              | 142               | 1300                 | 2200                 | 2900                 | 4400                 | 6200                   | 7100                  | 36                                | 30,803         | 0              | [outputs](outputs/investigation/2022-02-17T15:30:33) |  
+| PDFKit existing implementation (Run 1, fresh deployment) | 51.64               | 496              | 126               | 1200                 | 1400                 | 1700                 | 2400                 | 3300                   | 3600                  | 30.9                              | 30,988         | 0              | [outputs](outputs/investigation/2022-02-17T15:50:06) |  
+| PDFKit existing implementation (Run 2)                   | 52                  | 311              | 125               | 1200                 | 1400                 | 1600                 | 2600                 | 3100                   | 3100                  | 32                                | 31,200         | 1              | [outputs](outputs/investigation/2022-02-17T16:03:39) |  
+| PDFKit with custom CSS file (Run 1, fresh deployment)    | 51.49               | 558              | 191               | 1100                 | 1300                 | 1400                 | 2300                 | 2900                   | 3200                  | 30.9                              | 30,896         | 0              | [outputs](outputs/investigation/2022-02-18T11:43:31) |  
```

Comment:
The link for the outputs for this row and the one below are broken

Reply:
Fixed. The files were missing.